gef.py

```diff
@@ -538,6 +538,7 @@ class Elf:
     SPARC             = 0x02
     SPARC64           = 0x2b
     AARCH64           = 0xb7
+    RISCV             = 0xf3
```

Comment:
How much have you tested RISCV? Do you have a VM ready to use to test it?

Reply:
> How much have you tested RISCV?

Used a few simple examples (similar to the example shown at [Debugging with GDB]) to
verify `is_branch_taken` returned the correct result etc.

> Do you have a VM ready to use to test it?

[riscv-tools] bundles things up nicely, and includes [`riscv-isa-sim`] (`spike`) which enables emulating RISCV.

[Debugging with GDB]: https://github.com/riscv/riscv-isa-sim#debugging-with-gdb
[riscv-tools]: https://github.com/riscv/riscv-tools#riscv-tools-
[`riscv-isa-sim`]: https://github.com/riscv/riscv-isa-sim

Reply:
@hugsy I tested this out.See #310.

Reply:
Then it's a go for me.